test/unittests/model/version/test_version_range.py

```diff
@@ -33,6 +33,7 @@
     # pre-releases
     ['',                             [[[">=", "0.0.0-"]]],    ["1.0"],                ["1.0-pre.1"]],
     ['*, include_prerelease=True',   [[[">=", "0.0.0-"]]],    ["1.0", "1.0-pre.1", "0.0.0", "0.0.0-pre.1"],   []],
+    [', include_prerelease=True',   [[[">=", "0.0.0-"]]],    ["1.0", "1.0-pre.1", "0.0.0", "0.0.0-pre.1"],   []],
```

Comment:
I am not sure we want to support this syntax at all, ``requires = "dep/[,include_prerelease=True"`` looks really ugly and not intuitive what would be the expected behavior.

Reply:
I do agree, but wasn't sure whether adding an exception for this would be the preferred option. My assumption would be to only raise an exception when the version range specifier is empty and `include_prerelease` is specified. Leaving `"dep/[]"` as a valid range, but disallowing `"dep/[,include_prerelease]"`. Is this assumption correct?

Reply:
I see your point.

In retrospective, probably we shouldn't have allowed the empty ``[ ]`` in first place, it is possible that it slipped from previous implementations, or earlier unit tests. But you are right now that it is allowed, and we most likely don't want to risk breaking existing users that would rely on it, that it would be more consistent to allow the prereleases to work with the same syntax even if ugly.

So looks good, lets go for it.
Please check that you need to sign the CLA in order to be able to merge the PR. Many thanks for your contribution.

Reply:
Sounds good to me! I should have signed the CLA now and will push the discussed update to the test now.